server/plugin/virtual_agent.go

```diff
@@ -321,7 +321,7 @@ func (p *Plugin) CreateMessageAttachment(fileID string) (*MessageAttachment, err
 	var attachment *MessageAttachment
 	fileInfo, appErr := p.API.GetFileInfo(fileID)
 	if appErr != nil {
-		return nil, fmt.Errorf("error getting the file info. Error: %w", appErr)
+		return nil, fmt.Errorf("error getting the file info. Error: %s", appErr.Message)
```

Comment:
Why not `appErr.Error()`?

Reply:
If I use that the same thing happens i.e: two colons